src/main/java/uk/gov/hmcts/reform/em/stitching/rest/DocumentTaskResource.java

```diff
@@ -88,9 +79,10 @@ public DocumentTaskResource(DocumentTaskService documentTaskService) {
                     .body(result);
 
         } catch (RuntimeException e) {
-            log.error("Error while mapping entities for DocumentTask : {} ", docTask, e);
+            log.error("Error while mapping entities for DocumentTask : {} ", documentTaskDTO, e);
+            return ResponseEntity.badRequest().body(result);
```

Comment:
the idea is that you could throw an exception that has the required response code embedded, similar to this block in Annotations except ignore the use of Optional:


```
return optionalAnnotationSetDTO
            .map( annotationSetDTO -> ResponseEntity.ok(annotationSetDTO))
            .orElseThrow( () -> new ResourceNotFoundException("Could not find annotation set for this document id#" + documentId) );
```